XamarinCommunityToolkit/Views/AvatarView/ColorTheme.shared.cs

```diff
@@ -0,0 +1,141 @@
+﻿using System;
+using System.ComponentModel;
+using System.Linq;
+using Xamarin.Forms;
+using static System.Math;
+
+namespace Xamarin.CommunityToolkit.UI.Views
+{
+	[EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
Since we got rid of BaseColorTheme I think we should not hide ColorTheme (That comment was related to BaseColorTheme)

Please remove this line

Reply:
Sorry i hadn't realised the attribute was for the base class, I was a little confused myself 😛